### PR TITLE
20w16a jmx

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Comment by YanisBft** [comment]
Some javadocs wouldn't be too much, in a future pr if not now

File: mappings/net/minecraft/server/dedicated/ServerMBean.mapping [hunk 1]
```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5169 net/minecraft/server/dedicated/ServerMBean
```
Comment (modmuss50) [inline mappings/net/minecraft/server/dedicated/ServerMBean.mapping:1]: Humm, im not too sure about this name.

Comment (liach) [reply mappings/net/minecraft/server/dedicated/ServerMBean.mapping:1]: This is like a class that implements `DynamicMBean`, which the dynamic mbean supports retrieving information on server avg tick time and past tick times.

**Review by Runemoro — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/server/dedicated/ServerMBean.mapping [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	FIELD field_23970 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_23971 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_23972 beanInfo Ljavax/management/MBeanInfo;
+	FIELD field_23973 entries Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD getAttribute (Ljava/lang/String;)Ljava/lang/Object;
+		ARG 1 attribute
+	METHOD getAttributes ([Ljava/lang/String;)Ljavax/management/AttributeList;
+		ARG 1 attributes
```
Comment (YanisBft) [inline mappings/net/minecraft/server/dedicated/ServerMBean.mapping:17]: You named the setter parameters the same as that, but those ones aren't javax.management.Attribute. Maybe `attributeName` would be better?

Comment (liach) [reply mappings/net/minecraft/server/dedicated/ServerMBean.mapping:17]: these names are copied from jdk's names.

Comment (YanisBft) [reply mappings/net/minecraft/server/dedicated/ServerMBean.mapping:17]: fair

**Review by YanisBft — APPROVED** [review]

**Pull request merged**